CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Agglayer
+
+**🎉 Thank you for considering contributing to Agglayer!**
+We welcome contributions of all kinds, from code improvements to documentation fixes. This guide will help you understand how we collaborate and the conventions we follow.
+
+-----
+
+## 🚀 Getting Started
```

Comment:
First, we should define that we're using `conventional-commit` for our commits on main.
Then define that the branch and PR title should follow the same kind of mechanism

`feature/any-branch-name` and `feat: implement some cool stuff`

Reply:
The `pull request guidelines/commits` doesn't include the notion of `conventional commits` standard nor a link to it, I think it's better to add it for people to read the spec.

For the branches, I don't have a strong opinion about that, and most of the external contributor will use a fork that already contains their handle.